corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -187,6 +187,30 @@ def test_process_doc_from_sql(self, datetime_mock):
 
         CaseAccessorSQL.hard_delete_cases(case.domain, [case.case_id])
 
+    @patch('corehq.apps.userreports.specs.datetime')
+    @override_settings(TESTS_SHOULD_USE_SQL_BACKEND=True)
+    def test_process_filter_no_longer_pass(self, datetime_mock):
+        datetime_mock.utcnow.return_value = self.fake_time_now
+        sample_doc, expected_indicators = get_sample_doc_and_indicators(self.fake_time_now)
+
+        since = self.pillow.get_change_feed().get_current_offsets()
```

Comment:
@lwyszomi This recently changed in https://github.com/dimagi/commcare-hq/pull/15420 you'll need to merge master and change this line